guides/common/modules/proc_installing-google-gce-plugin.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Installing_Google_GCE_Plugin_{context}"]
+= Installing Google GCE Plugin
+
+Install the Google GCE plugin to attach an GCE compute resource provider to {Project}.
+This allows you to manage and deploy hosts to GCE.
+
+.Procedure
+. Install the Google GCE compute resource provider on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --enable-foreman-compute-gce
+----
+. Optional: In the {ProjectWebUI}, navigate to *Administer > About* and select the _compute resources_ tab to verify the installation of the Google GCE plugin.
```

Comment:
Use bold emphasis to format tab name?
And the capitalization of the tab name is consistent with the GUI?

Reply:
Fixed, cheched, and applied!